draft-private-access-tokens.md

```diff
@@ -173,14 +185,83 @@ Mediators advertise an Oblivious HTTP URI template {{!RFC6570}} for proxying
 protocol messages to Issuers. For example, one template
 for the Mediator might be https://mediator.net/relay-access-token-request.
 
-# Issuance
+# Protocol
+
+Private Access Tokens are single-use tokens cryptographically bound to
+policies. Redeemers request tokens from Clients, who then engage with
+Mediators and Issuers to private compute policy-compliant tokens and
+reveal them to the Redeemer. Example policies and use cases that system
+addresses are described in {{examples}}.
+
+The rest of this section describes this interactive protocol in terms of
+the redemption request ({{redemption-request}}) and corresponding token
+issuance flow ({{issuance}}).
+
+## Redemption Request
+
+The Client is assumed to have the policy verification key before redeeming
+a Private Access Token. See {{access-token-keys}} for details on how this
+can be obtained.
+
+Token redemption is an interactive protocol. Redeemers challenge Clients to
+present a unique, single-use token. Redeemers present this challenge to Clients
+with the following context:
+
+~~~
+struct {
+   uint8 policy_window;
+   opaque redeemer_name<1..2^16-1>;
```

Comment:
If this is bound the origin, this may be redundant—or we have it to ensure that the values match.